browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -30,7 +30,7 @@
  *     headers: [ { text: 'first column' }, { text: 'second' }],
  *     entries: [
  *         { row: 0, columns [ { text: 'a' }, { collapsed: 'collapsedIcon.svg' }, { collapsedimage: '<BASE64 encoded PNG>' } ] },
- *         { row: 1, columns [ { text: 'a2' }, { expanded: 'expandedIcon.svg' }, selected: true ]}
+ *         { row: 1, columns [ { link: 'http://someurl' }, { expanded: 'expandedIcon.svg' }, selected: true ]}
```

Comment:
Below address is used for this purpose, just fyi.
https://example.com/

Reply:
done